src/net/socket_utils.zig

```diff
@@ -18,74 +19,344 @@ pub const PACKETS_PER_BATCH: usize = 64;
 // The identifier for the scoped logger used in this file.
 const LOG_SCOPE: []const u8 = "socket_utils";
 
-pub const SocketThread = struct {
-    allocator: Allocator,
-    handle: std.Thread,
+const XevThread = struct {
+    pub const Handle = std.Thread.ResetEvent;
 
-    pub fn spawnSender(
+    const List = std.DoublyLinkedList(struct {
         allocator: Allocator,
```

Comment:
Does it make sense to have an allocator per-node?

Reply:
SocketThreads which make the Nodes could be created with diffferent allocators